quantum/process_keycode/process_unicode_common.c

```diff
@@ -124,6 +136,12 @@ __attribute__((weak)) void unicode_input_finish(void) {
             break;
         case UC_WIN:
             unregister_code(KC_LALT);
+#ifdef UNICODE_UC_WIN_USE_NUMPAD_IF_POSSIBLE
+            if (!unicode_saved_num_lock) {
+                // We switched numlock on, so switch it off again.
+                tap_code(KC_NUMLOCK);
```

Comment:
Prefer comments that describe _why_ the code does something rather than _how_ it does it (see below).

```suggestion
                tap_code(KC_NUMLOCK);
```